src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,79 @@
+import React, { useCallback } from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageClick = useCallback((page: number) => {
+    if (page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  }, [onPageChange, totalPages]);
+
+  const handlePrevClick = useCallback(() => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  }, [currentPage, onPageChange]);
+
+  // Функція для кліку на наступну сторінку
+  const handleNextClick = useCallback(() => {
+    if (currentPage < totalPages) {
+      onPageChange(currentPage + 1);
+    }
+  }, [currentPage, onPageChange, totalPages]);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <button
+          className="page-link"
+          data-cy="prevLink"
+          aria-disabled={currentPage === 1}
+          onClick={handlePrevClick}
+          disabled={currentPage === 1}
```

Comment:
You're using both `aria-disabled` and `disabled` on the button element, which is good for accessibility. However, the `aria-disabled` attribute is unnecessary when you're using the `disabled` attribute because `disabled` already conveys that the button is not interactive to assistive technologies.

Reply:
fixed